### DISCUSSION TITLE
Add canonical url  support

### DISCUSSION THREAD

**Discussion by mattw0110** [avitorio/outstatic#171] [Ideas]
Add a self-referencing canonical URL  and an option in the writer to assign another URL if needed.

**Comment by avitorio** [comment]
Hey @mattw0110, thanks for the suggestion. 

The canonical url should be set on the [frontend](https://stackoverflow.com/questions/75938020/how-to-set-canonical-tag-in-nextjs-13).

For custom canonical urls, you could create a custom field and check if it has been populated. If not just use the actual page url.

Let me know if this works or if you need any help with it.

**Reply by mattw0110** [reply]
Worked Great!
